src/fileWriters/templatizeFile.js

```diff
@@ -0,0 +1,20 @@
+import fse from 'fs-extra';
+import yaml from 'js-yaml';
+
+const templatizeFile = async ({ fileName, filePath }) => {
+  let fileContents = fse.readFileSync(filePath, 'utf8');
+  if (fileName === 'package.json') {
+    fileContents = JSON.parse(fileContents);
+    fileContents.name = '{{packageName}}';
+    fileContents.repository = 'https://github.com/edx/{{packageName}}';
+    fileContents.version = '0.1.0';
```

Comment:
Why is this version `0.1.0`

Reply:
Because this is creating a new app, we should reset whatever the current version is in `front-end-cookie-cutter-application`. The current version in that repo was `0.1.0`, so I just moved that here. Do you have a suggestion of what version makes more sense?

Reply:
Yeah, I think `1.0.0` makes the most sense.